src/documents/practices/DesignForPrivacy.html.md.handlebars

```diff
@@ -0,0 +1,29 @@
+---
+title: Design for Privacy
+layout: content
+category: Practices
+---
+
+Many people rely on technology and “smart” services to provide accessibility in their everyday lives. For example, devices that allow voice commands to control lighting or other appliances in the home can significantly improve the accessibility of an interior space. However, the use of these services come with the cost and associated risk of sharing personal information online. Those who can benefit most from these smart services, including persons with disabilities, persons who are aging and others who face discrimination, stereotyping, or exclusion, are often the most vulnerable to the misuse of private information - for example through denial of medical insurance, jobs and services, or fraud.
```

Comment:
This paragraph is a bit dense. It's possible to split it into two paragraphs, the first being the statement, the second being the example.

"More and more people are using technology and smart services to provide accessibility in their everyday lives. However, using these services can create a risk of unintended sharing of personal information to other parties. Therefore, those who can benefit most from these smart services, like persons with disabilities, persons who are aging, or people who face discrimination, are often the most vulnerable to the misuse of private information.

For example, devices that allow voice commands to control lighting or other appliances in the home can significantly improve the accessibility of an interior space, but could inadvertently expose personal information that could be used to deny medical insurance, jobs and services, or increase the risk of fraud."